native/packages/react-native-bpk-component-chip/src/BpkChipDismissable.js

```diff
@@ -0,0 +1,101 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import React from 'react';
+import { View, ViewPropTypes, StyleSheet, type StyleObj } from 'react-native';
+import PropTypes from 'prop-types';
+import BpkText from 'react-native-bpk-component-text';
+import {
+  borderRadiusSm,
+  colorGray100,
+  colorGray700,
+  spacingSm,
+  spacingMd,
+} from 'bpk-tokens/tokens/base.react.native';
+import BpkChipDismissButton from './BpkChipDismissButton';
+
+const styles = StyleSheet.create({
+  wrapper: {
+    alignItems: 'center',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    padding: spacingSm,
+    backgroundColor: colorGray100,
+    borderRadius: borderRadiusSm,
+  },
+  wrapperLarge: {
+    paddingVertical: spacingMd,
+  },
+  text: {
+    color: colorGray700,
+  },
+  dismissBtn: {
+    marginLeft: spacingSm,
+  },
+});
+
+type Props = {
+  dismissButtonLabel: string,
+  label: string,
+  large: boolean,
+  onDismiss: ?() => mixed,
+  style: ?StyleObj,
+};
+
+const BpkChipDismissable = (props: Props) => {
```

Comment:
Should this be `BpkDismissableChip`? That sounds a lot better to me. Do we have a naming convention that dictates this?

Reply:
It certainly does sound better that way. I'll see if we have a convention.